mobile/cmp/input/Select.js

```diff
@@ -288,7 +288,6 @@ class Model extends HoistInputModel {
             if (action === 'input-change') {
                 this.inputValue = value;
                 this.inputValueChangedSinceSelect = true;
-                if (!value) this.noteValueChange(null);
```

Comment:
do we have any sense of what we were trying to accomplish with noteValueChange(null)?? IIRC we were trying to restore null when user when back to empty string.  Does not seem to be an issue after this change, however.